.idea/kotlinc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wasn't this supposed to either be ignored or in the update kotlin PR?

Reply:
My bad, i will remove it